installer/service_manager.py

```
@@ -7,12 +7,14 @@
 import platform
 import re
 from pathlib import Path
+from docker_setup import DockerSetup
```

Pylint (3.3.7)

[error] Unable to import 'docker_setup'

(E0401)